packer/scylla_install_image

```diff
@@ -111,18 +111,17 @@ if __name__ == '__main__':
         run(f'curl -L -o /tmp/amazon-ssm-agent.deb https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/debian_{deb_arch()}/amazon-ssm-agent.deb', shell=True, check=True)
```

Comment:
Out of curiosity, do we delete this .deb file after installation? (has nothing to do with this patch though!)